### DISCUSSION TITLE
how can i get interaction response text with func:on_message?

### DISCUSSION THREAD

**Discussion by WandererLu** [Rapptz/discord.py#9411] [Q&A]
i cant get bot interaction response content with on_message,im not sure if i shoud use other functions

**Comment by lightlybakedman** [comment]
Why are you trying to get the interaction response text with on_message? What are you actually trying to achieve?

**Reply by WandererLu** [reply]
My own bot cannot read the content of other bot's interactions (which might be called embed messages) sent to users. the content is what I want to achieve

**Reply by lightlybakedman** [reply]
> My own bot cannot read the content of other bot's interactions

Why? Because you don't have the message_content intent? using on_message won't change anything
